bigtop-packages/src/common/impala/install_impala.sh

```diff
@@ -154,12 +154,12 @@ function symlink_lib() {
     rm $file
     base=`basename $file`
     versionless=${base/-[0-9].*/.jar}
-    ln -s ../../$dir/`basename $versionless` $DEPENDENCY_DIR/
+    ln -s $dir/`basename $versionless` $DEPENDENCY_DIR/
 }
 # Remove MR1 Hive shim
 rm -f $DEPENDENCY_DIR/hive-shims-0.23*.jar $DEPENDENCY_DIR/hadoop-core*.jar;
-for file in $DEPENDENCY_DIR/libhdfs*.so*; do symlink_lib $file ../${NATIVE_LIB_DIR}; done
-for file in $DEPENDENCY_DIR/libhadoop*.so*; do symlink_lib $file hadoop/lib/native; done
+for file in $DEPENDENCY_DIR/libhdfs*.so*; do symlink_lib $file /usr/hdp/current/hadoop-client/../usr/lib; done
```

Comment:
Ah that makes sense!

It would have been nice if `/usr/hdp/current` was just a plain symlink to a `/usr/hdp/2.3.0.0-2557` that had everything under it. Oh well.

Thanks for the explanation!